ivre/xmlnmap.py

```diff
@@ -199,6 +199,33 @@ def ignore_script(script):
     return False
 
 
+def cpe2dict(cpe_str):
+    """Helper function to parse CPEs. This is a very partial/simple parser."""
+    # Remove prefix
+    if len(cpe_str) < 5 or cpe_str[:5] != "cpe:/":
+        sys.stderr.write("WARNING, invalid cpe format (%s)\n" % cpe_str)
+        return {"value": cpe_str}
+    cpe_body = cpe_str[5:]
+    parts = cpe_body.split(":", 3)
+    l = len(parts)
```

Comment:
Please use a longest (and more understandable) variable name, such as `nparts` for example.